lib/cn/route.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
+ */
+
+struct route_map;
+struct kvs_cparams;
+
+struct route_map *
+route_map_create(const struct kvs_cparams *cp, const char *kvsname);
```

Comment:
I like `edge_map` better than `route_map`...

Reply:
i can change it..

Reply:
or maybe route is better.  i dunno.  i'm on the edge.

Reply:
Very punny!   Easier to change now before everyone starts writing to it, I'm good either way..